.github/workflows/ci.yaml

```diff
@@ -17,6 +18,127 @@ jobs:
       - name: Run ah lint
         working-directory: ./charts/mlrun-ce
         run: ah lint
+
+  ci-runner:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Extract private key
+      run: echo "${{ secrets.MLRUN_PEM }}" > mlrun.pem
+    - name: Set permissions on key
+      run: chmod 400 mlrun.pem
+    - name: Install sshpass
+      run: sudo apt-get install -y sshpass
+    - name: Copy pem to remote machine
+      run: |
+        sudo scp -r -o StrictHostKeyChecking=no -i mlrun.pem  /home/runner/work/ce ${{ secrets.USERNAME }}@${{ secrets.IP }}:/home/${{ secrets.USERNAME }}/ce
+    - name: Cleaning instance
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        # CLEANUP
+        echo "helm uninstall mlrun-ce"
+        helm uninstall -n mlrun mlrun-ce || echo true
+        echo "helm uninstall monitoring"
+        helm uninstall -n mlrun monitoring || echo true
+        echo "delete pv, pvc from mlrun"
+        kubectl delete all --all -n mlrun
+        echo "deleting mlrun namepsace";
+        kubectl delete namespace mlrun || echo true
+        EOF
+        
+    - name: Creating mlrun namespace
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        echo "create mlrun namepsace"
+        kubectl create namespace mlrun        
+        echo "k8s namespaces - $(kubectl get namespace)"
+        EOF
+        
+    - name: Creating docker registry secret
+      run: |
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        kubectl --namespace mlrun create secret docker-registry registry-credentials \
+        --docker-server ${{ secrets.DOCKER_SERVER }} \
+        --docker-username ${{ secrets.DOCKER_USERNAME }} \
+        --docker-email=${{ secrets.DOCKER_EMAIL }} \
+        --docker-password=${{ secrets.DOCKER_PASSWORD }}
+        EOF
+
+    - name: CE installation
+      run: |    
+        # CE INSTALLATION
+        sshpass ssh -o StrictHostKeyChecking=no -i  mlrun.pem ${{ secrets.USERNAME }}@${{ secrets.IP }} <<EOF
+        echo $(pwd)
+        cd ce/ce/charts/mlrun-ce
+        echo $(ls)
+        echo "helm dependency update"
+        helm dependency update
+        echo "installing community edition"
+        helm install mlrun-ce --namespace mlrun \
+        --set minio.resources.requests.memory="128Mi" \
+        --set  jupyterNotebook.persistence.size="1Gi" \
+        --set  mlrun.db.persistence.size="1Gi" \
+        --set  mlrun.api.persistence.size="1Gi" \
+        --set  mpi-operator.deployment.create="true" \
+        --set  mlrun.httpDB.dbType="sqlite" \
+        --set  mlrun.httpDB.dirPath="/mlrun/db" \
+        --set  mlrun.httpDB.dsn="sqlite:////mlrun/db/mlrun.db?check_same_thread=false"  \
+        --set  mlrun.httpDB.oldDsn=""  \
```

Comment:
This was only for running CE in GitHub worker (mysql didn't work). Remove all these https values so the tests indeed run with mysql